west.yml

```diff
@@ -46,6 +48,10 @@ manifest:
       path: modules/fs/fatfs
       groups:
         - fs
+    - name: fff
+      remote: meekrosoft
+      revision: pull/102/head
+      path: modules/lib/fff
```

Comment:
I wouldn't do that.  Why don't you just point to your repo or wait til either
1. merged upstream
2. forked under zephyrproject-rtos

Just in case if you want to point to your repo:
```yaml
    - name: fff
      url: https://github.com/yperess/fff
      revision: master
      path: modules/lib/fff
```

Reply:
https://docs.zephyrproject.org/latest/guides/modules.html#process-for-submitting-changes-to-existing-modules

Reply:
Didn't know! Thanks.